types/node.lua

```diff
@@ -662,7 +878,12 @@ function nodetypes.vbox:unbox ()
    return { self }
 end
 
-function nodetypes.vbox:append (box)
+--- Added a box or several to the current vbox.
+-- The box height is the height is the total height of the content, minus the depth of the last box.
+-- The depth of the vbox is the depth of the last box.
+--
+-- @tparam box|table box A box or a list of boxes to add to the vbox.
+function vbox:append (box)
```

Comment:
So the parameter is sometimes a single box, sometimes a vboxlist. Unclear API?

It _could_ take both, but the check below is that if it is a box, it can be unboxed. Lot's of undocumented assumptions.

- "insertions" use a vboxlist (material to split) or a box (via its own vbox-derived class).
- If not mistaken, the only other use case is `pagebuilder.collateVboxes` with a vboxlist. This method by the way has _nothing_ to do with the pagebuilder. It's a fairly dumb convenience, and see above comment on the constructor.